2023/05/27 05:37:28 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "gii". in C:\Apache24\htdocs\demo\yii\framework\web\CController.php:483
Stack trace:
#0 C:\Apache24\htdocs\demo\yii\framework\web\CController.php(270): CController->missingAction('gii')
#1 C:\Apache24\htdocs\demo\yii\framework\web\CWebApplication.php(282): CController->run('gii')
#2 C:\Apache24\htdocs\demo\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/gii')
#3 C:\Apache24\htdocs\demo\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\Apache24\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/gii
---
2023/05/27 05:40:20 [error] [exception.CDbException] CDbException: CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder.
 in C:\Apache24\htdocs\demo\yii\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\Apache24\htdocs\demo\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\Apache24\htdocs\demo\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\Apache24\htdocs\demo\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\Apache24\htdocs\demo\yii\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\Apache24\htdocs\demo\protected\controllers\SiteController.php(30): CModule->__get('db')
#5 C:\Apache24\htdocs\demo\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 C:\Apache24\htdocs\demo\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\Apache24\htdocs\demo\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 C:\Apache24\htdocs\demo\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 C:\Apache24\htdocs\demo\yii\framework\web\CWebApplication.php(282): CController->run('')
#10 C:\Apache24\htdocs\demo\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#11 C:\Apache24\htdocs\demo\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 C:\Apache24\htdocs\demo\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/demo/index.php
---
2023/05/27 05:40:41 [error] [exception.CDbException] CDbException: CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder.
 in C:\Apache24\htdocs\demo\yii\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\Apache24\htdocs\demo\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\Apache24\htdocs\demo\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\Apache24\htdocs\demo\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\Apache24\htdocs\demo\yii\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\Apache24\htdocs\demo\protected\controllers\SiteController.php(30): CModule->__get('db')
#5 C:\Apache24\htdocs\demo\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 C:\Apache24\htdocs\demo\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\Apache24\htdocs\demo\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 C:\Apache24\htdocs\demo\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 C:\Apache24\htdocs\demo\yii\framework\web\CWebApplication.php(282): CController->run('')
#10 C:\Apache24\htdocs\demo\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#11 C:\Apache24\htdocs\demo\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 C:\Apache24\htdocs\demo\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/demo/index.php
---
2023/05/27 05:41:11 [error] [php] count(): Parameter must be an array or an object that implements Countable (C:\Apache24\htdocs\demo\yii\framework\gii\generators\model\ModelCode.php:371)
Stack trace:
#0 C:\Apache24\htdocs\demo\yii\framework\gii\CCodeGenerator.php(162): ModelCode->prepare()
#1 C:\Apache24\htdocs\demo\yii\framework\gii\CCodeGenerator.php(63): ModelGenerator->prepare()
#2 C:\Apache24\htdocs\demo\yii\framework\web\actions\CInlineAction.php(49): ModelGenerator->actionIndex()
#3 C:\Apache24\htdocs\demo\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 C:\Apache24\htdocs\demo\yii\framework\web\CController.php(286): ModelGenerator->runAction()
#5 C:\Apache24\htdocs\demo\yii\framework\web\CController.php(265): ModelGenerator->runActionWithFilters()
#6 C:\Apache24\htdocs\demo\yii\framework\web\CWebApplication.php(282): ModelGenerator->run()
#7 C:\Apache24\htdocs\demo\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#8 C:\Apache24\htdocs\demo\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 C:\Apache24\htdocs\demo\index.php(13): CWebApplication->run()
REQUEST_URI=/demo/index.php?r=gii/model/index
in C:\Apache24\htdocs\demo\index.php (13)
2023/05/27 05:42:50 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "list". in C:\Apache24\htdocs\demo\yii\framework\web\CController.php:483
Stack trace:
#0 C:\Apache24\htdocs\demo\yii\framework\web\CController.php(270): CController->missingAction('list')
#1 C:\Apache24\htdocs\demo\yii\framework\web\CWebApplication.php(282): CController->run('list')
#2 C:\Apache24\htdocs\demo\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('config/list')
#3 C:\Apache24\htdocs\demo\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\Apache24\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=config/list
---
2023/05/27 05:43:15 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "admin". in C:\Apache24\htdocs\demo\yii\framework\web\CController.php:483
Stack trace:
#0 C:\Apache24\htdocs\demo\yii\framework\web\CController.php(270): CController->missingAction('admin')
#1 C:\Apache24\htdocs\demo\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#2 C:\Apache24\htdocs\demo\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/admin')
#3 C:\Apache24\htdocs\demo\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\Apache24\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/admin
---
